beacon-chain/state/interfaces.go

```diff
@@ -115,7 +116,7 @@ type ReadOnlyValidator interface {
 	WithdrawableEpoch() primitives.Epoch
 	ExitEpoch() primitives.Epoch
 	PublicKey() [fieldparams.BLSPubkeyLength]byte
-	WithdrawalCredentials() []byte
+	GetWithdrawalCredentials() []byte
```

Comment:
Why the change?

Reply:
It needs to follow the interface

Reply:
the `WithWithdrawalCredentials` interface? I think that interface's function can also be called `WithdrawalCredentials` and it shouldn't cause an issue

Reply:
the protobuf doesn't have WithdrawalCredentials as a function, only GetWithdrawalCredentials